libs/sdk-ui-ext/src/internal/components/configurationControls/ConfigSection.tsx

```diff
@@ -69,7 +69,7 @@ export class ConfigSection extends React.Component<IConfigSectionProps, IConfigS
         const configSectionClassName = `adi-bucket-configuration ${className}`;
 
         return (
-            <div className={this.getSectionClassNames()}>
+            <div className={this.getSectionClassNames()} aria-label="Configuration section">
```

Comment:
While this fine for now, there really should be localized as well. But do not change it now :)